packages/stencil/src/components/datacom-radio/datacom-radio.tsx

```diff
@@ -1,10 +1,11 @@
-import { Component, Host, h, Prop, Element, EventEmitter, Event, VNode, State, Method } from '@stencil/core';
+import { Component, h, Prop, Element, EventEmitter, Event, VNode, State, Method, Host } from '@stencil/core';
 import { getSvg } from '../../common/images/icon-provider';
 import { randomString } from '../../utils';
 import { FormControl } from '../form-control';
 
 export type RadioSize = 'standard' | 'small';
-export type RadioVariant = 'radios' | 'buttons';
+export type RadioVariant = 'radios' | 'buttons' | 'grouped';
```

Comment:
Should rather just use 'radio' and 'button'. No need for plurals.

Also, you don't need this grouped variant? We introduced all that state work so we didn't have to have an extra variant, right?